### DISCUSSION TITLE
Management UI - TypeError

### DISCUSSION THREAD

**Discussion by ian-1964** [rabbitmq/rabbitmq-server#10843] [Other] [answered]
### Describe the bug

![image](https://github.com/rabbitmq/rabbitmq-server/assets/164875534/52e92990-3dcd-40bf-b4d7-981dc8ac3f5d)

This error appears in Google Chrome at the bottom of the management webpage when one of Exchanges, Queues or Admin tabs are selected. 

Had to go back to version 3.10.14 before this error did not occur. All versions after this that I tried, produced the error to varying degrees on one or more tabs.

### Reproduction steps

1. Deploy container
2. Log in to management webpage using Google Chrome
3. Select Exchanges, Queues or Admin tab

### Expected behavior

I would not expect to see an error message.

### Additional context

Environment: Rancher for Windows v1.12.3
Kubernetes: v1.29.2
Docker engine: moby
Container: rabbitmq:3-management-alpine (v3.13.0)

The error is not apparent when using the Microsoft Edge browser

**Comment by michaelklishin** [comment, marked as answer]
This has been discussed to death before. Clear your browser's cache [after upgrading](https://www.rabbitmq.com/docs/upgrade#management-plugin-upgrades).

**Reply by ian-1964** [reply]
Thank you. 
It may have been done to death before, but I've not come across it. 
So, please check your attitude at the door. There is no need for it!
